lib/ecto/schema.ex

```diff
@@ -195,6 +198,7 @@ defmodule Ecto.Schema do
       import Ecto.Schema, only: [schema: 2]
       @primary_key {:id, :integer, read_after_writes: true}
       @timestamps_type Ecto.DateTime
+      @timestamps_use_usec true
```

Comment:
Sorry, I see your comments now that you decided to go with true. :) Don't worry, let me think about this more and I will merge it later.